### ISSUE TITLE
Invalid syntax in tap

### ISSUE THREAD

**Issue by farzadmf** [zix99/rare#67]
Hey,

Just tried to install from Homebrew and I get an error:
```
❯ brew tap zix99/rare
==> Tapping zix99/rare
Cloning into '/usr/local/Homebrew/Library/Taps/zix99/homebrew-rare'...
remote: Enumerating objects: 45, done.
remote: Counting objects: 100% (45/45), done.
remote: Compressing objects: 100% (30/30), done.
remote: Total 45 (delta 14), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (45/45), 6.65 KiB | 3.33 MiB/s, done.
Resolving deltas: 100% (14/14), done.
Error: Invalid formula: /usr/local/Homebrew/Library/Taps/zix99/homebrew-rare/rare.rb
rare: Calling bottle :unneeded is disabled! There is no replacement.
Please report this issue to the zix99/rare tap (not Homebrew/brew or Homebrew/core):
  /usr/local/Homebrew/Library/Taps/zix99/homebrew-rare/rare.rb:9

Error: Cannot tap zix99/rare: invalid syntax in tap!
```

Is there anything I need to do other than
```
brew tap zix99/rare && brew install rare
```

I'm doing this on a 2019 Mac Book Pro (Intel)
Thank you

**Comment by zix99** [comment]
Thanks for the report.  I believe I've fixed the issue and need to release a new version.  I'm in the process of cleaning up and merging #66 , and will release a new version after that.

**Comment by farzadmf** [comment]
Great, looking forward to that

**Comment by zix99** [comment]
Rather than waiting for me to finish the merge, I released a fix with 0.2.2 and tested it on a mac. Let me know if it works for you, and if so, I'll close this issue

**Comment by farzadmf** [comment]
Great, thanks for letting me know!

Tried it and it seems to be working, I'm gonna close this issue.

Just want thing: not sure if it's significant or not, but I see the following warning:
```
Warning: Tried to install empty array to /usr/local/Cellar/rare/0.2.2/share/man/man1
```

Just wanted to let you know 🙂
